.env.example

```diff
@@ -5,14 +5,34 @@
 # Local Development Configuration
 # =============================================================================
 
-# Database URL for local development
+# Database Configuration
 DATABASE_URL=postgresql+asyncpg://admin:password@localhost:5432/ai_virtual_agent
+POSTGRES_DB=ai_virtual_agent
+POSTGRES_USER=admin
+POSTGRES_PASSWORD=password
+POSTGRES_PORT=5432
```

Comment:
Do we need to pass them again when we have Database_url?

Reply:
Added comment to further explain this. The URL is used by the App and POSTGRES_* vars are used by Podman Compose/containers.